pkg/nfd-master/nfd-master.go

```diff
@@ -57,6 +59,8 @@ import (
 	"sigs.k8s.io/node-feature-discovery/pkg/version"
 )
 
+const DefaulMaxNumberOfConcurrentLabelers = 200
```

Comment:
I think 200 is kinda big number for default concurrent labelers, maybe we can set it to `10` ?